frontend/apps/reader/readerui.lua

```diff
@@ -595,6 +595,8 @@ end
 function ReaderUI:showReader(file, provider, seamless, is_provider_forced)
     logger.dbg("show reader ui")
 
+    file = ffiUtil.realpath(file)
+
     if lfs.attributes(file, "mode") ~= "file" then
```

Comment:
Also it doesn't work as written:

```
./luajit: frontend/apps/reader/readerui.lua:600: bad argument #1 to 'attributes' (string expected, got nil)
stack traceback:
```

Reply:
Not invalid, but if the file doesn't exist for example realpath will return nil.

Reply:
I forgot about that, thanks.

Reply:
No longer relevant; a new approach is being taken.